File: src/test/java/de/whitefrog/neobase/test/Likes.java
package de.whitefrog.neobase.test;

import de.whitefrog.neobase.model.relationship.BaseRelationship;

public class Likes extends BaseRelationship<Person, Person> {
  private String field;

  public String getField() {
    return field;
  }

  public void setField(String field) {
    this.field = field;
  }
}

File: base/src/test/java/de/whitefrog/frogr/test/model/Likes.kt
package de.whitefrog.frogr.test.model

import de.whitefrog.frogr.model.relationship.BaseRelationship

class Likes @JvmOverloads constructor(from: Person, to: Person, var field: String? = null) : 
  BaseRelationship<Person, Person>(from, to)
